metadata_layer.py

```diff
@@ -1093,7 +1102,7 @@ def mirror_link(self, pool, data_source, link, link_obj, policy):
             if pool and pool.edition and pool.edition.title:
                 title = pool.edition.title
             else:
-                title = None
+                title = self.title or None
```

Comment:
Will we be running this code on the books we've already imported? I was thinking those would continue to work, but anything new that we imported would be different.